drivers/dide/inst/templates/task_run.sh

```diff
@@ -6,10 +6,9 @@ echo hipercow version: {{hipercow_version}}
 echo hipercow.dide version: {{hipercow_dide_version}}
 echo running on: $(hostname -f)
 
-export PATH=/opt/apps/lmod/lmod/libexec:$PATH
-source /opt/apps/lmod/lmod/init/bash
-export LMOD_CMD=/opt/apps/lmod/lmod/libexec/lmod
-module use /modules-share/modules/all
+source /etc/profile
+
+module use /modules/modules/all
```

Comment:
As above - we do this for all R jobs - this one is the template for running a normal cluster job; the one above is similar but specifically runs jobs to provision packages for the user.